core/domain/domain.en.yml

```diff
@@ -1310,33 +1238,30 @@ responses:
   utter_test_navigation__explanations_4:
     - text: I will provide information about referrals, as well as other details, such as appointment policy, for each available testing site in your area
 
-  utter_ask_test_navigation__continue_no_assessment:
+  utter_ask_test_navigation__continue:
     - text: Do you want to continue and search for a testing site?
+
+  utter_ask_test_navigation__continue_error_offer_assessment:
+    - text: Sorry, do you want to continue and search for a testing site?
       buttons:
-        - title: Yes continue
-          payload: /continue
+        - title: Yes
+          payload: "/affirm"
```

Comment:
Is there a reason to have some payloads with quotes and others without?

Reply:
There's a reason but it's not a good one. Both are valid we just didn't chose a format (no quotes except when necessary vs always put quotes anyway) and ended up with both.

Reply:
Ok :shrug: